src/main/java/uk/gov/hmcts/darts/annotation/service/impl/AnnotationUploadServiceImpl.java

```diff
@@ -44,13 +48,16 @@ public Integer upload(MultipartFile multipartFile, Annotation annotation) {
         var containerLocations = annotationDataManagement.upload(binaryData, multipartFile.getOriginalFilename());
 
         var annotationEntity = annotationMapper.mapFrom(annotation);
+        annotationRepository.save(annotationEntity);
+
         var checksum = fileContentChecksum.calculate(binaryData.toBytes());
         var annotationDocumentEntity = annotationDocumentBuilder.buildFrom(multipartFile, annotationEntity, checksum);
+        annotationDocumentRepository.save(annotationDocumentEntity);
```

Comment:
Why do we need to persist here? the persistence happens at the end the AnnotationPersistenceService

Reply:
Because when the annotation is pushed to arm the persist and merge was causing a proxy error when you add an existing annotation document to a new EOd, so when I removed the persist and merge I needed to make sure all teh objects were saved.

Reply:
Saving here breaks the cleaning up of documents on the db failure.  Persistence calls needs to go in the persistence layer for this work.  Will raise another ticket to fix.